src/git.py

```diff
@@ -121,8 +121,10 @@ def _set_trailers(self, patch: Patch) -> str:
         # Add trailers, changing them if they exist.
         # It's _highly_ unlikely that they'd exist, but this seems to be the
         # most sane way of handling that edge case.
-        with tempfile.NamedTemporaryFile(mode='wt') as f:
+        with tempfile.NamedTemporaryFile(mode='w+') as f:
+          f.seek(0, os.SEEK_END)
```

Comment:
I'm not sure what this seek does/if it's necessary.
Perhaps a comment is in order?

Reply:
I could have sworn that it didn't work without it, but I just tried it without the seek and it appears to work.

Will fix.

Reply:
Done.